templates/base.html

```diff
@@ -23,6 +24,8 @@
                     <li class="py-2"><a href="glossary_report.html" class="hover:text-gray-300">Glossary</a></li>
                     <li class="py-2"><a href="web_report.html" class="hover:text-gray-300">Generated Web Pages</a></li>
                     <li class="py-2"><a href="config.html" class="hover:text-gray-300">Config</a></li>
+                    <li class="py-2"><a href="adt_a11y_report.html" class="hover:text-gray-300">Virtual Screen Reader smoke test</a></li>
+                    <li class="py-2"><a href="adt_aria_at_report.html" class="hover:text-gray-300">ARIA-AT &amp; WCAG assessment</a></li>
```

Comment:
Minor but can we think of something shorter / catchier / more obvious? Do these need to be two separate reports or can they be on the same page?